ts/create-kpt-functions/tsconfig.json

```diff
@@ -1,19 +1,26 @@
 {
   "compilerOptions": {
-    "module": "commonjs",
-    "target": "es6",
-    "resolveJsonModule": true,
+    "allowUnreachableCode": false,
+    "allowUnusedLabels": false,
+    "baseUrl": "node_modules",
```

Comment:
ditto

Reply:
Removed baseUrl configs in all packages, in this PR and for demo-functions